backend/app/literature/crud/editor_crud.py

```diff
@@ -50,15 +50,15 @@ def destroy(db: Session, editor_id: int) -> None:
 
 
 def patch(db: Session, editor_id: int, editor_update: EditorSchemaCreate) -> dict:
-
+    editor_data = jsonable_encoder(editor_update)
     editor_db_obj = db.query(EditorModel).filter(EditorModel.editor_id == editor_id).first()
     if not editor_db_obj:
         raise HTTPException(status_code=status.HTTP_404_NOT_FOUND,
                             detail=f"Editor with editor_id {editor_id} not found")
     res_ref = stripout(db, editor_update, non_fatal=True)
     add(res_ref, editor_db_obj)
 
-    for field, value in editor_update.dict().items():
+    for field, value in editor_data.items():
```

Comment:
I believe @dougli1sqrd added in the .dict() when he was doing the large MyPy pull request. I thought it was not necessary. You should consult with him to make sure he agrees.

Reply:
If mypy checks out (which it seems like it does) and this runs just fine then this is fine by me. I think I was previously getting errors that items() is not a function on that type. But this seems fine by me!